docs/java-rest/high-level/miscellaneous/x-pack-usage.asciidoc

```diff
@@ -0,0 +1,54 @@
+[[java-rest-high-x-pack-usage]]
+=== X-Pack Usage API
+
+[[java-rest-high-x-pack-usage-execution]]
+==== Execution
+
+Detailed information about the usage of features from {xpack} can be
+retrieved using the `xpackUsage()` method:
```

Comment:
The name isn't quite right here. I suspect the name in `xpackInfo` isn't right either now that I see this. I bet I didn't catch that in the rename....

Reply:
I think the name is correct. What do you think it should be?

Reply:
Ah ic, xpackUsage is the helper method in RequestConverters.